packages/@aws-cdk/core/lib/private/runtime-info.ts

```diff
@@ -26,6 +28,11 @@ export function collectRuntimeInformation(): cxschema.RuntimeInfo {
         foundMatch = true;
       }
     }
+    for (const pkg of WHITELIST_PACKAGES) {
```

Comment:
Use `WHITELIST_PACKAGES.includes()` instead

Reply:
Yeah, that would be smarter. On it...